sklearn/decomposition/_lda.py

```diff
@@ -278,6 +278,23 @@ class LatentDirichletAllocation(TransformerMixin, BaseEstimator):
         Prior of topic word distribution `beta`. If the value is None, it is
         `1 / n_components`.
 
+    See Also
+    --------
+    sklearn.discriminant_analysis.LinearDiscriminantAnalysis:
+        A classifier with a linear decision boundary, generated by fitting
+        class conditional densities to the data and using Bayes’ rule.
+
+    References
+    ----------
+    .. [1] "Online Learning for Latent Dirichlet Allocation", Matthew D.
+        Hoffman, David M. Blei, Francis Bach, 2010
+
+    [2] "Stochastic Variational Inference", Matthew D. Hoffman, David M. Blei,
+        Chong Wang, John Paisley, 2013
+
+    [3] Matthew D. Hoffman's onlineldavb code. Link:
```

Comment:
I think,

```suggestion
    .. [2] "Stochastic Variational Inference", Matthew D. Hoffman, David M. Blei,
        Chong Wang, John Paisley, 2013

    .. [3] Matthew D. Hoffman's onlineldavb code. Link:
```

but need to check the rendering (`ci/circleci: doc artifact` CI job), currently it doesn't look ideal: https://142780-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.decomposition.LatentDirichletAllocation.html

Reply:
It might raise an error if the article is not linked (somewhere we expect `[2]_`) in the docstring. If this is not the case, we could either link where it is meaningful, otherwise, we could as well remove the reference.